### ISSUE TITLE
Please don't shuffle your data.

### ISSUE THREAD

**Issue by munkh-erdene** [hanazahrah/Forex-Prediction-using-SVR#1]
Write it shuffle=False on train_test_split

**Comment by hanazahrah** [comment]
ah yes, my bad. Thank you for point it
